CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Migration docs for migration from v2 to v3 of the testing suite.
 
 ### Changed
+- [BREAKING] The composer.json configurations `config.youwe-testing-suite.type` and `config.mediact-testing-suite.type`
```

Comment:
The reason for this change could be shared here as well: "This was invalid as the config section belongs to configuration from Composer itself. Moving it to the extra section which is intended for this purpose (composer docs: "arbitrary extra data for consumption by scripts")"